src/plugins/lineendings/README.md

```diff
@@ -0,0 +1,13 @@
+- infos = Information about the lineendings plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = 
+- infos/placements = pregetstorage presetstorage precommit 
+- infos/description =
+
+## Introduction ##
+
+The Lineendings Plugin verifies the Lineendings of a file.
+If inconsistent lineendings or lineendings that don't match `config/valid` are detected the plugin yields an error.
```

Comment:
valid could be set to CR, LF or CRLF (string value) to tell the plugin to reject all other lineendings. without the valid key, only inconsistent lineendings are rejected
gonna update the readmes